bin/agentConsole.js

```diff
@@ -27,7 +27,7 @@ var readline = require('readline'),
     iotAgentLib = require('../lib/fiware-iotagent-lib'),
     commandUtils = require('command-node'),
     config = require('../config'),
-    logger = require('fiware-node-logger'),
+    logger = require('logops'),
```

Comment:
I'll try it more closely in the https://github.com/telefonicaid/context-adapter since probably I am not doing things the right way... ;) Thanks!